<launch>

    <!-- Launch GPS Manager -->
    <node name="gps_manager" pkg="decision_igvc" type="gps_manager" output="screen">
        <rosparam param="at_goal_tolerance">0.5</rosparam>
        <rosparam param="waypoints">
        [
        49.2626396517, -123.248471077,
        49.2624439204, -123.248579687
        ]
        </rosparam>
        <rosparam param="base_frame">base_link</rosparam>
        <rosparam param="global_frame">map</rosparam>
    </node>

</launch>
